src/index.html

```diff
@@ -2,15 +2,37 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+          content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+          href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="/"
+          class="form form--big"
+          method="get"
+          data-qa="big">
```

Comment:
```suggestion
    <form
      action="/"
      class="form form--big"
      method="get"
      data-qa="big"
    >
```

Reply:
Pay attention to formatting. Fix similar problem in all code
<img width="992" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/86789161/e624ec1e-34f1-409a-89d2-ae7fa15e5188">